chemicaltoolbox/openbabel/ob_prepare_ligands.xml

```diff
@@ -0,0 +1,86 @@
+<tool id="openbabel_prepare_ligands_2" name="Prepare Docking Ligands" version="0.1.0">
+    <description>Tool to prepare ligands for docking with tools like Autodock Vina. OpenBabel is used to perform the conversion.</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements"/>
+    <command detect_errors="aggressive"><![CDATA[
+	    mkdir output && obabel -i ${ligands.ext} -o $oformat -O output/molecule.$oformat -m $ligands $gen3d -p $pH_value
```

Comment:
Please single-quote all data and text parameters. Security reasons and if whitespaces are in a path ... you never know who is deploying these tools. :)

Reply:
Done

Reply:
Done